tests/components/enphase_envoy/conftest.py

```diff
@@ -36,13 +36,13 @@ def config_fixture():
     }
 
 
-@pytest.fixture(name="gateway_data", scope="session")
+@pytest.fixture(name="gateway_data")
```

Comment:
I'd say that speed is more important than memory for the tests. How many times are these fixtures used?

Reply:
Can we use the `"package"` scope?

Reply:
Not used often – happy to switch to `"package"`.